File: ch4/exo2.py
#!/usr/bin/env python3

## print a summary of how many times each word was seen
print("Enter a list words: ")

words = []
while True:
    try:
        words.append(input())
    except:
        break

freq = dict.fromkeys(words, 0)

for w in words:
    freq[w] += 1

print()
for w in sorted(freq):
    print("{} => {}".format(w, freq[w]))

File: ch4/exo2.pl
#!/usr/bin/env perl
use strict;
use warnings;
use utf8;

## print a summary of how many times each word was seen

print("Enter a list of words: \n");

chomp(my @words = <STDIN>);

my %freq;

foreach (@words) {
	++$freq{$_};
}

print("\n");
foreach (sort keys %freq) {
	print("$_ => $freq{$_}\n");
}

